app/helpers/reference_helper.rb

```diff
@@ -0,0 +1,38 @@
+module ReferenceHelper
+
+  def after_desired_date?
+    Time.zone.today > Date.parse('2015-12-31')
+  end
+
+  def table_header
+    applicant = t('processed_applications.table_header.applicant')
+
+    if after_desired_date?
+      reference = t('processed_applications.table_header.reference')
+      "<th> #{reference} </th>\n<th> #{applicant} </th>".html_safe
+    else
+      "<th> #{applicant} </th>".html_safe
+    end
+  end
+
+  def table_column(application)
+    prefix = "<td> <a href=\"/#{controller.controller_name}/#{application.id}\">"
```

Comment:
In theory this method can be rewritten as:

```ruby
  def table_column(application)
    if after_desired_date?
      "<td> #{link_to(application.reference, url_for(controller: controller.controller_name, action: :show, id: application.id))} </td>\n<td> #{application.applicant} </td>".html_safe
    else
      "<td> #{link_to(application.applicant, url_for(controller: controller.controller_name, action: :show, id: application.id))} </td>".html_safe
    end
  end
```

Which looks cleaner.

Whereas my solution, aesthetically unpleasant, it's already functional with the tests passing. 
(And we'll be scrapping the whole thing in three weeks.)

But then we introduce the following:

* the tests now complain about: 

```
     ActionController::UrlGenerationError:
       No route matches {:action=>"show", :controller=>"test", :id=>1}
```
* what do I do about the line lengths for RuboCop?

Also, how is my solution insecure? Just curious, because the controller and the application argument is something the app will pass to the helper, they won't be something obtained from the user.